build.yml

```diff
@@ -0,0 +1,26 @@
+sdk: '2'
+language: bash
+go-runtime:
+  version: '1.10-alpine'
+native:
+  image: 'openjdk:8-jre-alpine'
+  static:
+      - path: src/main/sh/native.sh
+        dest: native
+  build:
+    image: 'openjdk:8-slim'
+    deps:
+      - 'apt-get update && apt-get install -y make bash wget gradle'
+    run:
+      - 'make'
+      - 'gradle shadowJar'
+    artifacts:
+      - path: '/native/build/libs/native-jar-with-dependencies.jar'
+  test:
+    image: 'openjdk:8-jre-alpine'
```

Comment:
Because some comments above you noted on a line of the "test" section:

"__Tests are running on top of openjdk:8-slim, according to the config__"

You didnt answered my question as reply of that comment so I understood that I should have added it to the test section.